sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2048,6 +2048,16 @@ object SQLConf {
     .checkValue(threshold => threshold > 0, "The threshold must be a positive integer.")
     .createWithDefault(1024)
 
+  val INPUT_CODEGEN_RDD_INTERRUPT_ON_CANCEL =
+    buildConf("spark.sql.codegen.inputCodegenRDD.interruptOnCancel")
+      .internal()
+      .doc("Whether to interrupt InputCodegenRDD on the task cancellation. Interrupt " +
+        "InputCodegenRDD would be useful to stop task and release resource quickly, " +
+        "and also reduce the task killing timeout failures by task reaper.")
+      .version("3.5.0")
```

Comment:
Could you update this value, @Ngone51 ? Maybe, `4.1.0` because this is an `Improvement`?

<img width="706" alt="Screenshot 2025-03-13 at 20 53 34" src="https://github.com/user-attachments/assets/c732cd86-17c0-4cb2-9db1-d46a071bd108" />